src/pages/Payment.vue

```diff
@@ -52,6 +56,7 @@
 <script>
 import Store from '../store.js';
 import Instructions from '../components/Instructions.vue';
+import NetMgr from '../services/netMgr.js';
```

Comment:
Store includes netmgr, you can do Store.netMgr but this is by no means mandatory!